### PR TITLE
Suppress urllib3 InsecureRequestWarnings when `validate_certs` option is false

### PR DESCRIPTION
**Pull request by qazbnm456**
##### SUMMARY

It's clear that the user would know the possible risk when he or she chose to turn off the option, so the warning message could be ignored and make the output clean.

##### ISSUE TYPE

- Refactoring Pull Request

##### COMPONENT NAME

`community.general.proxmox`

### CODE REVIEW

**Review by felixfontein — APPROVED** [review]
Thanks for your contribution! Could you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

**Comment by qazbnm456** [comment]
Done!

File: changelogs/fragments/5915-suppress-urllib3-insecure-request-warnings.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - proxmox - Suppress urllib3 InsecureRequestWarnings when validate_certs option is false.
```
Comment (felixfontein) [inline changelogs/fragments/5915-suppress-urllib3-insecure-request-warnings.yml:2]:
```suggestion
  - proxmox - suppress urllib3 ``InsecureRequestWarnings`` when ``validate_certs`` option is ``false`` (https://github.com/ansible-collections/community.general/pull/5931).
```

**Review by felixfontein — COMMENTED** [review]
I have a small update for the changelog fragment :) Can you also look at the failing sanity tests? From the GitHub diff it is not obvious what the problem is, are there maybe some invisible Unicode characters or Windows line endings (though I think that these would result in a different error message)?

**Comment by qazbnm456** [comment]
All done :)

**Review by felixfontein — COMMENTED** [review]
Looks good to me. I'll merge if nobody objects in the next week.

**Comment by felixfontein** [comment]
@qazbnm456 thanks for your contribution!
@russoz thanks for reviewing!
